neutralino.js/package.json

```
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "jest",
```

as per standardjs' official doc, would it be better to just do:
```suggestion
    "test": "standard && jest",
```